packages/cli/src/spec/DoctorSpec.ts

```diff
@@ -24,6 +24,24 @@ import {MockPrompt} from './mock/MockPrompt';
 import {enUS as messages} from '../lib/strings';
 
 describe('doctor', () => {
+  let originalProcess: NodeJS.Process;
+
+  beforeEach(() => {
```

Comment:
So I guess this at least makes it explicit that we don't have any doctor tests for Mac?

Reply:
Yeah, `validatePath()` already handles other platforms, I think.